packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -315,10 +315,7 @@ class TestConfiguration {
       roles,
     })
     await cache.user.invalidateUser(globalId)
-    return {
-      ...resp,
-      globalId,
-    }
+    return resp
```

Comment:
Why doesn't think include the `globalId` anymore?

Reply:
It wasn't used by anything (tests seem to pass without it there), and it was preventing the `User` type from being the correct return type of this function.

Reply:
Ah perfect in that case!